CMakeLists.txt

```diff
@@ -724,6 +724,14 @@ if (VAST_ENABLE_CORELIGHT_CAT)
           DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif ()
 
+option(VAST_ENABLE_DBSIZE_SH "Install dbsize.sh with VAST" ON)
```

Comment:
We should probably rename that script to `vast-du-pcent.sh` or so, and add an explanatory comment to inside the script given that we now install it.

Reply:
I renamed it to `vast-df-percent.sh` and modified it for portability.